index.css

```diff
@@ -0,0 +1,467 @@
+
+    /* left: 16px;
+  margin-top: -250px;
+  top: 3620px; */
+    margin-top: -300px;
+    height: 70vh;
+    width: 100%;
+    flex-direction: column;
+    flex-wrap: wrap;
+    background: #1a2236;
+    border: 0.333333px solid rgba(255, 255, 255, 0.1);
+    backdrop-filter: blur(1px);
+  }
+
+  .project-name-6 {
+    /* position: absolute; */
+    display: flex;
+    align-items: center;
+
+    /* left: 16px;
+  top: 4175px; */
+    margin-top: -300px;
+    height: 70vh;
+    width: 100%;
+    flex-direction: column;
+    flex-wrap: wrap;
+    background: #1a2236;
+    border: 0.333333px solid rgba(255, 255, 255, 0.1);
+    backdrop-filter: blur(1px);
+  }
+
+  h3 {
+    font-style: normal;
+    font-weight: 300;
+    font-size: 20px;
+    line-height: 80%;
+    color: #fff;
+  }
+
+  .languages {
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+    justify-content: space-around;
+  }
+
+  .html {
+    background: #98dede;
+    margin: 10px;
+    padding: 10px;
+    width: 87px;
+    height: 20px;
+  }
+
+  .rails {
+    background: #98dede;
+    margin: 10px;
+    padding: 10px;
+    width: 87px;
+    height: 20px;
+  }
+
+  .js {
+    background: #98dede;
+    margin: 10px;
+    padding: 10px;
+    width: 87px;
+    height: 20px;
+  }
+
+  .see-this-project {
+    display: flex;
+    align-self: flex-start;
+    width: 178px;
+    height: 44px;
+    border: 1px solid #fff;
+    margin-left: 50px;
+    color: white;
+    margin-top: 50px;
+  }
+
+  .text {
+    text-align: center;
+    font-weight: bolder;
+    margin-left: 10px;
+  }
+
+  .arrow {
+    width: 20.57px;
+    height: 15.35px;
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 17px;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .abt {
+    font-size: 50px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 50px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #1a2236;
+  }
+
+  .detail {
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 21px;
+    letter-spacing: 0;
+    text-align: left;
+    color: #1a2236;
+  }
+
+  .get-my-cv {
+    width: 100%;
+    height: 289px;
+    background: #1c528a;
+  }
+
+  h4 {
+    font-size: 20px;
+    font-style: normal;
+    font-weight: 800;
+    line-height: 30px;
+    letter-spacing: 0;
+    text-align: center;
+    color: #fff;
+  }
+
+  ul li {
+    font-size: 18px;
+    font-style: normal;
+    font-weight: 600;
+    line-height: 27px;
+    letter-spacing: -0.0525em;
+    text-align: center;
+    color: #fff;
+  }
+
+  .framework {
+    width: 100%;
+    height: 289px;
+    background: #e2505c;
+  }
+
+  .skills {
+    width: 100%;
+    height: 289px;
+    background: #febf52;
+  }
+
+  .cv {
+    width: 450px;
+    height: 80px;
+    margin-left: 5px;
+    background-color: #1a2236;
+    margin-bottom: 50px;
+  }
```

Comment:
Your styling implements most of the template design. However, the rules applied here affects the margin in this section and is causing your page to overflow. Here is a screenshot to help you fix it. Also, kindly use the font specified in the template for this section

<img width="715" height="370" alt="about" src="https://user-images.githubusercontent.com/71434803/125151066-6264e980-e13c-11eb-9b8a-2fed2e1795de.png">